### ISSUE TITLE
Korean pocket-tts model

### ISSUE THREAD

**Issue by seastar105** [kyutai-labs/pocket-tts#261]
Thanks for updating training code!

I could train korean model, and seems working.

- **100M student:** https://huggingface.co/seastar105/pocket-tts-korean-100m
- **300M teacher:** https://huggingface.co/seastar105/pocket-tts-korean-300m

Student models have quite poor performance while teacher is quite usable one.

## Usage

```bash
  uvx pocket-tts generate \
      --config hf://seastar105/pocket-tts-korean-100m/korean.yaml \
      --voice ./voice_prompt.wav \
      --text "안녕하세요. 한국어 음성 합성 모델입니다."
```

 ## Training

I trained the models using this Korean dataset:
https://huggingface.co/datasets/seastar105/emilia-yodas-ko-filtered-pocket-tts

The dataset contains around 2k hours(900k samples) from yodas2-ko. And i used qwen3-forced-aligner to get alignment. 

When I trained model from scratch, it collapsed fast at the beginning of the train(~1,2k steps). So I warm-started the 300M teacher from English 24-layer checkpoint. 

Samely, I initialized the 100M student from the English 6-layer checkpoint and depth-distilled it from the Korean teacher. It has much advantage.

Both models are trained with 50k steps with 4 x RTX 5090 cards, code is [here](https://github.com/seastar105/pocket-tts-korean-training/tree/main/training)

I archived all intermediate checkpoints here:
- https://huggingface.co/seastar105/pocket-tts-checkpoints

## Evaluation

I evaluated all 25 EMA checkpoints from each training run on all 500 Korean zero-shot items from yuekai/CV3-Eval `zero_shot_ko`.

CER metric gets low very fast that flats while UTMOS kepp going high during training teacher model, while student training UTMOS,CER getting worse

<img width="1400" height="900" alt="Image" src="https://github.com/user-attachments/assets/fb267a5d-740e-4a9e-bcf2-d29d5aed1dbb" />

<img width="1400" height="900" alt="Image" src="https://github.com/user-attachments/assets/bf91c68d-ae15-4a7d-ae6a-9f1a165c60c4" />

## NOTE

These models are very poor at english, numbers reading, while acoustic quality and speaker sim is quite good. I think it's reasonable as dataset size is not large and training steps are small. 

Poor student model performance may get better if model starts from teacher-init and training longer.

**Comment by manukyutai** [comment]
Hello, amazing work, thanks a lot!

If you want your models to be featured in the readme (alongside the Czech model), we'd love to let the community know that this exists.
Out of curiosity, how many it/s did you get during training with your setup?

For the distillation, it's sometimes indeed a bit tricky. Things you could try if you haven't already:

- Init with the teacher weights but shrank, but since this is the default behavior you may have tried this already
- Init with the english student but take the text embedding tables from your teacher or re-init the tables since you're using a different text tokenizer (if you didn't retrain the tokenizer you probably should)
- If you cannot distill in 6, you can distill in more. Distilling in 24 is still worth it cause you at least have the CFG baked in.

Let me know if I can help

**Comment by seastar105** [comment]
@manukyutai 
> If you want your models to be featured in the readme (alongside the Czech model), we'd love to let the community know that this exists.
Out of curiosity, how many it/s did you get during training with your setup?

happy to see thus model gets featured

for training throughput, i got ~2.6it/s in teacher training and ~3.0it/s for student training

both used batch size 8, grad accum 2

> Init with the teacher weights but shrank, but since this is the default behavior you may have tried this already
> Init with the english student but take the text embedding tables from your teacher or re-init the tables since you're using a different text tokenizer (if you didn't retrain the tokenizer you probably should)
> If you cannot distill in 6, you can distill in more. Distilling in 24 is still worth it cause you at least have the CFG baked in.

second option was exactly what i used to train student

when compared first two options with short steps training(10k) second option showed much better. so i just choose second one.

using bit more layers looks good option thanks for advice

**Comment by gabrieldemarmiesse** [comment]
> happy to see thus model gets featured 

Feel free to open a pull request to modify the readme. I encourage users to list only TTS that are "usable". So in your case, maybe the 300m only.

**Comment by gabrieldemarmiesse** [comment]
Hi @seastar105 would it be ok for you to list this TTS in our readme?

**Comment by seastar105** [comment]
@gabrieldemarmiesse 
I made PR to add in README. could you check it?
